uaa/slateCustomizations/source/index.html.md.erb

```diff
@@ -378,6 +378,29 @@ _Response Fields_
   Added in UAA 3.3.0
 </aside>
 
+When an access token is revoked using this endpoint, the UAA Introspect Token endpoint (``/introspect``)
+will respond with ``"active": false`` when presented with the revoked access token.
+
+If the access token is in the JWT format (as opposed to the opaque format), the server config ``uaa.jwt.revocable`` or
+the Identity Zone config ``config.tokenPolicy.jwtRevocable`` must be set to ``true`` for
+this feature to work. However, OAuth resource servers are generally not required to call the UAA Introspect
```

Comment:
> resource servers are generally not required to call the UAA Introspect

They are *never* required to call the introspect endpoint. It's just that when they have an opaque token, they don't have an option. And when they have a jwt one, they shouldn't, though they may.

Reply:
Point discussed and/or edits made.